thoth/storages/graph/models.py

```diff
@@ -51,6 +51,12 @@
 )
 
 
+association_table = Table('association', Base.metadata,
```

Comment:
let's follow some example we have in the databse like: https://github.com/thoth-station/storages/blob/e6fbbc34f782b8555729cf9f8fe270b2753f341f/thoth/storages/graph/models.py#L355